lib/matplotlib/text.py

```diff
@@ -2092,7 +2093,9 @@ def draw(self, renderer):
                 self.arrow_patch.figure = self.figure
             self.arrow_patch.draw(renderer)
 
-        Text.draw(self, renderer)
+        astext = copy.copy(self)
+        astext.__class__ = Text
```

Comment:
Note that by "problem" I don't mean that bbox_artist() is broken, just that some assumption somewhere is broken at that point.